compiler/codegen/cg-type.cpp

```diff
@@ -93,58 +93,91 @@ void PrimitiveType::codegenDef() {
 void ConstrainedType::codegenDef() {
 }
 
-#ifdef HAVE_LLVM
-static void codegenFunctionTypeWideLlvm(FunctionType* ft) {
-  // Use a 'int64' dynamic index. Do not bother generating a function type.
-  auto t = llvm::Type::getInt64Ty(gContext->llvmContext());
-  if (!ft->symbol->hasLLVMType()) {
-    gGenInfo->lvt->addGlobalType(ft->symbol->cname, t, false);
-    ft->symbol->llvmImplType = t;
-    ft->symbol->llvmAlignment = ALIGNMENT_DEFER;
+static void codegenFunctionTypeWide(FunctionType* ft) {
+  if (auto outfile = gGenInfo->cfile) {
+    // TODO: Should I get the code-generated 'int(64)' type here instead?
+    fprintf(outfile, "typedef %s %s;\n\n", "int64_t", ft->symbol->cname);
```

Comment:
I think you should solve the TODO

Should be `dtInt[INT_SIZE_64]->codegen().c.c_str()`

Reply:
Yeah, that should work, but the `//TODO` is mainly there because I'm not really sure what the precedent is. Do we care to do that? Because we don't really seem to do it consistently, we'll just generate `int64_t` and `void*` etc in other places. What's your preference here?

Reply:
My preference would be an approach that has both precedent and means that "int64_t" only shows up in our code once for codegen. I pulled that snippet from `VarSymbol::codegenDef`, but as long as there is precedent I am good with it